src/game/player.py

```diff
@@ -35,11 +35,19 @@ def reset(self):
     def is_alive(self):
         return self.hp > 0
 
+    def get_click_value(self):
+        item_click_value = sum(item.increase if item.bought else 0 for item in self.upgrades['Click'].values())
+        return self.base_click_value + item_click_value
+
+    def get_income(self):
+        upgrade_income = sum(item.increase if item.bought else 0 for item in self.upgrades['Income'].values())
```

Comment:
Random question: what does it mean for a Player to have an item where `item.bought` is `False`?

Reply:
Oh - is that that each player will have all the items, but they won't initially be bought?